src/components/structures/MessagePanel.js

```diff
@@ -135,14 +136,13 @@ export default class MessagePanel extends React.Component {
         // whether to show reactions for an event
         showReactions: PropTypes.bool,
 
-        // whether to use the irc layout
-        useIRCLayout: PropTypes.bool,
+        // which layout to use
+        layout: Layout,
```

Comment:
Hmm, does passing a TypeScript type directly like this actually work...? I assume React is expect one of the `PropTypes` here instead... (Really we should convert this file to TypeScript and do away with the prop types since TS types can be used in place of them, but that should be in a separate PR.)